src/Http/Controllers/Admin/ModuleController.php

```diff
@@ -474,7 +474,10 @@ public function edit($id, $submoduleId = null)
             return View::exists($view);
         });
 
-        return View::make($view, $this->form($id));
+        // @todo: So this is not really ideal, maybe we can use
+        // views::share or something to make the required data available
+        // to the blade components?
+        return View::make($view, $this->form($id) + ['form' => $this->form($id)]);
```

Comment:
Actually I think this is fine, but we can also have 2 parameters in the blade component, one for the fields and one for the item.